### ISSUE TITLE
New content element: essay

### ISSUE THREAD

**Issue by cjcodeproj** [cjcodeproj/vtmedia-schema#28]
The `<essay>` element is a proposed element structure for short written works.  It's the first attempt at coverage of cataloging written copy.  The first examples will be written work that may be included in some copies of DVD media.  The presence of a `<booklet>` element under the `media/medium/productSpecs/inventory` element hierarchy would suggest that there are printed materials included with the DVD or Blu-Ray that can be documented.

The `<essay>` element structure should record the title of the work, any copyright information, authorship, and maybe additional information identifying the length of the work (word count, column inches?)

The branch associated with this issue will have a rudimentary tag structure in place, and examples where appropriate.

Depends on: https://github.com/cjcodeproj/vtmedia-schema/issues/24 (completed)

**Comment by cjcodeproj** [comment]
New element has been merged into main branch.

https://github.com/cjcodeproj/vtmedia-schema/pull/30
